prdoc/pr_4091.prdoc

```diff
@@ -0,0 +1,13 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Remove `pallet::getter` usage from authority-discovery pallet
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removes `pallet::getter`s from `pallet-authority-discovery`.
+      The syntax `StorageItem::<T, I>::get()` should be used instead.
+
+crates:
+  - name: pallet-authority-discovery
```

Comment:
you also need to provide a `bump` item here, please check the schema

Reply:
Ah, good looks. I've added the appropriate bump now 👍